src/fides/data/sample_project/sample_resources/sample_systems.yml

```diff
@@ -13,8 +13,7 @@ system:
       progress: Complete
       link: https://example.org/cookie_house_dpia
     privacy_declarations:
-      - name: Analyze customer behaviour for improvements.
```

Comment:
Removed the `name` here as we're deprecating that on privacy declarations, and instead the Admin UI shows the descriptive name of the `data_use` itself.